crates/holochain/Cargo.toml

```diff
@@ -70,6 +70,8 @@ rand = "0.8.5"
 rand_chacha = "0.3.1"
 rand-utf8 = "0.0.1"
 rusqlite = { version = "0.32.1" }
+# Used to set a crypto provider for TLS connections.
+rustls = { version = "0.23.25", features = ["ring"] }
```

Comment:
The default feature is `aws_lc`. Arguably we could use that already.

Reply:
I think we should. it looks like `ring` isn't going to be maintained and we're already having all these problems with building `aws-lc`, we may as well just start using it